backend/auth/role_map_repository.go

```diff
@@ -122,42 +220,36 @@ func addMatrix(m1 map[string]map[string]map[models.OperationType]struct{},
 		m1, m2 = m2, m1
 	}
 
-	ogns1 := make([]string, 0, len(m1))
-	for namespace := range m1 {
-		if _, exists := m2[namespace]; !exists {
-			ogns1 = append(ogns1, namespace)
-		}
-	}
-	ogres1 := make([]string, 0, len(m1["*"]))
-	for resource := range m1["*"] {
-		if _, exists := m2["*"][resource]; !exists {
-			ogres1 = append(ogres1, resource)
-		}
-	}
+	res := deepCopy(m1)
```

Comment:
This var name don't say much to me. Why matrix change to resource (or maybe it isnt resource)

Reply:
Maybe result? But we shouldn’t have to guess.